manifests/yamls/argocd.yaml

```diff
@@ -2437,7 +2437,7 @@ spec:
       - command:
         - /shared/argocd-util
         - rundex
-        image: quay.io/dexidp/dex:v2.25.0
+        image: "quay.io/dexidp/dex:v2.25.0"
         imagePullPolicy: Always
```

Comment:
Let's also make `imagePullPolicy: IfNotPresent` wherever it is set `Always` that will un-necessary download of the image.

Reply:
Okay